docs/docs-content/profiles/cluster-profiles/create-cluster-profiles/define-profile-variables.md

```diff
@@ -129,6 +138,13 @@ this guide from step three at the **Profile Layers** stage of cluster profile cr
 
     ![Palette YAML editor with the added profile variables.](/profiles_create-cluster-profiles_define-profile-variables_add-vars-to-yaml.webp)
 
+    :::tip
+
+    To improve navigation, you can change the display order of variables. Select the **Variables Three-dot menu** >
```

Comment:
We don't use `>` to indicate sub-menus. It's something we commonly fix when accepting contributions as it makes it hard for those using screen readers.